### PR TITLE
APMSPII-403 - added files for status endpoint

### PR DESCRIPTION
**Pull request by paulwalkington**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <Properties/>
+    <FaultResponse>
+        <Set>
+            <Headers/>
+            <Payload contentType="text/plain"/>
+            <StatusCode>401</StatusCode>
+            <ReasonPhrase>Server Error</ReasonPhrase>
+        </Set>
+        <AssignVariable>
+            <Name>pds.error.code</Name>
```
Comment (Sourceless) [inline proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml:13]: This is probably too PDS specific for this repo

File: proxies/live/apiproxy/policies/ServiceCallout.CallSpineHealthcheck.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ServiceCallout async="false" continueOnError="true" enabled="true" name="ServiceCallout.CallSpineHealthcheck">
+    <DisplayName>ServiceCallout.CallSpineHealthcheck</DisplayName>
```
Comment (Sourceless) [inline proxies/live/apiproxy/policies/ServiceCallout.CallSpineHealthcheck.xml:3]: Would be better to make this more generic - it won't always be a spine healthcheck

File: proxies/live/apiproxy/resources/jsc/SetStatusResponse.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+var apiproxy_revision = context.getVariable('apiproxy.revision');
+
+var spine_response_code = context.getVariable('spineHealthcheckResponse.status.code');
```
Comment (Sourceless) [inline proxies/live/apiproxy/resources/jsc/SetStatusResponse.js:3]: This could also do with being much more generic, spine won't necessarily be involved

**Review by Sourceless — CHANGES_REQUESTED** [review]

File: proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml (hunk 1 above)
Comment (paulwalkington) [reply proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml:13]: shall i just make this a simple generic fault?

File: proxies/live/apiproxy/policies/ServiceCallout.CallSpineHealthcheck.xml (hunk 2 above)
Comment (paulwalkington) [reply proxies/live/apiproxy/policies/ServiceCallout.CallSpineHealthcheck.xml:3]: I can change the name of this

File: proxies/live/apiproxy/resources/jsc/SetStatusResponse.js (hunk 3 above)
Comment (paulwalkington) [reply proxies/live/apiproxy/resources/jsc/SetStatusResponse.js:3]: i'll change the name of this

File: proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml (hunk 1 above)
Comment (paulwalkington) [reply proxies/live/apiproxy/policies/RaiseFault.401Unauthorized.xml]: removed pds varaibles

**Review by Sourceless — APPROVED** [review]
LGTM

**Review by alexhawdon — CHANGES_REQUESTED** [review]

**Review by alexhawdon — APPROVED** [review]

**Pull request merged**